src/topology/metric_space/isometry.lean

```diff
@@ -30,130 +30,137 @@ between pseudoemetric spaces, or equivalently the distance between pseudometric
 def isometry [pseudo_emetric_space α] [pseudo_emetric_space β] (f : α → β) : Prop :=
 ∀x1 x2 : α, edist (f x1) (f x2) = edist x1 x2
 
+/-- On pseudometric spaces, a map is an isometry if and only if it preserves nonnegative
+distances. -/
+lemma isometry_emetric_iff_nndist [pseudo_metric_space α] [pseudo_metric_space β] {f : α → β} :
```

Comment:
The name looks uselessly verbose to me, especially since there is absolutely no `emetric` in sight. What about `isometry_iff_nndist_eq` (and the next one could be renamed `isometry_iff_dist_eq`).